touchforms/backend/touchcare.py

```diff
@@ -248,25 +256,32 @@ def getIDsForValue(self, field_name, value):
 
 class CCInstances(InstanceInitializationFactory):
 
-    def __init__(self, sessionvars, api_auth):
+    def __init__(self, sessionvars, api_auth, form_context):
         self.vars = sessionvars
         self.auth = api_auth
         self.fixtures = {}
+        self.form_context = form_context or {}
 
     def generateRoot(self, instance):
         ref = instance.getReference()
-    
         def from_bundle(inst):
             root = inst.getRoot()
             root.setParent(instance.getBase())
             return root
 
         if 'casedb' in ref:
-            return CaseInstanceTreeElement(instance.getBase(), 
-                        CaseDatabase(self.vars.get('host'), self.vars['domain'], self.vars['user_id'], 
-                                     self.auth, self.vars.get("additional_filters", {}),
-                                     self.vars.get("preload_cases", False)),
-                        False)
+            return CaseInstanceTreeElement(
+                instance.getBase(),
+                CaseDatabase(
+                    self.vars.get('host'),
+                    self.vars['domain'],
+                    self.auth,
```

Comment:
This used to be the call to init:
```
def __init__(self, host, domain, user, auth, additional_filters=None, preload=False):		
       super(CaseDatabase, self).__init__(host, domain, auth, additional_filters, preload)
```
it completely ignores the user parameter and just calls the super method